SOURCES/xenvm-bugtool2.xml

```diff
@@ -1,3 +1,4 @@
 <collect>
-<directory pattern=".*">/etc/xenvm.d</directory>
+<directory pattern=".*">/var/run/nonpersistent/xenvm.d</directory>
+<command label="xenvm_diagnostics">/opt/xensource/libexec/xenvm_diagnostics</command>
```

Comment:
ah, perhaps this dropped off when the .spec file was moved from xen-dist-ocaml.hg.

Thanks!